modules/network/polygon.ts

```diff
@@ -204,6 +204,18 @@ const polygonNetworkData: NetworkData = {
                 path: 'value',
                 isIbYield: true,
             },
+            overnightUSDPlus: {
+                tokenAddress: '0x5d9d8509c522a47d9285b9e4e9ec686e6a580850',
+                sourceUrl: 'https://api.overnight.fi/optimism/usd+/fin-data/avg-apr/week',
+                path: 'value',
+                group: 'OVERNIGHT',
+            },
+            overnightstUSDPlus: {
```

Comment:
same token different address?

Reply:
No, this is the stUSD+, the other one is the USD+

Reply:
ah didn't catch that 👍